### ISSUE TITLE
swagger.json and Postman collection are not consitent /user vs /users

### ISSUE THREAD

**Issue by rupertlssmith** [gothinkster/realworld#380]
In the swagger definition, fetching the current user is done with a GET /users:

https://github.com/gothinkster/realworld/blob/master/api/swagger.json#L108

In the Postman test collection, fetching the current user is done with a GET /user:

https://github.com/gothinkster/realworld/blob/master/api/Conduit.postman_collection.json#L224

**Comment by jaguililla** [comment]
I ran into the same problem. What is the endpoint ot use?

Also, the update user allows to change the username (which can affect articles authors, etc.) and also allows you to change the token (should that be stored in any place?).

Thanks for your help!

**Comment by anishkny** [comment]
Good catch. We are going with:

```
Get Current User - GET /api/user
Update User - PUT /api/user
```

See: https://github.com/gothinkster/realworld/blob/master/api/README.md#get-current-user

Feel free to send a PR to correct `swagger.json`

**Comment by jaguililla** [comment]
Ok. I'll do the PR :)

Could tell me if PUT is supporting all fields? For me, `username` and `token` shouldn't be changed (token is not allowed in readme, so I assume that it is wrong in the Swagger spec, right?).

Thanks again!

**Comment by anishkny** [comment]
Yes. PUT user supports only `email, bio, image`

**Comment by jaguililla** [comment]
Done... If something is wrong tell me in #396 :)

**Comment by jaguililla** [comment]
The `POST {{APIURL}}/profiles/celeb_{{USERNAME}}/follow` Postman route has a body which seem redundant (as the username is already a path parameter). Would it be OK to delete the body of this request from the Postman collection?

**Comment by borovikovd** [comment]
Hi all. Faced the same problem, the fix is here: #664

**Comment by geromegrignon** [comment]
fixed by #664
